release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Runtime.Intrinsics.md

```diff
@@ -0,0 +1,974 @@
+# System.Runtime.Intrinsics
```

Comment:
System.Runtime.Intrinsics LGTM

This is namely the removal of the `where T : struct` constraint for easier use in generic contexts (this does not expand the set of supported `T`) and the addition of the `Vector512<T>` type

CC. @dotnet/area-system-runtime-intrinsics